lib/pf/activation.pm

```diff
@@ -477,12 +477,13 @@ Send an email with the activation code
 sub send_email {
     my ($type, $activation_code, $template, %info) = @_;
     my $logger = get_logger();
+    my $lang = $info{lang};
     my $profile = pf::Connection::ProfileFactory->_from_profile($info{portal}) // pf::Connection::ProfileFactory->_from_profile($DEFAULT_PROFILE);
 
     my $user_locale = clean_locale(setlocale(POSIX::LC_MESSAGES));
-    if ($type eq $SPONSOR_ACTIVATION) {
+    if ($lang) {
         $logger->debug('We are doing sponsor activation', $user_locale);
```

Comment:
This line isn't true anymore since it checks the presence of $lang and not the fact we're doing sponsor.

Just adjust the logging and we're golden

Reply:
Done in ae10236cf8a3da511ab9fe710b5389a94c91f0a1